cmd/youtubedr/download.go

```diff
@@ -41,15 +43,20 @@ func download(id string) error {
 
 	log.Println("download to directory", outputDir)
 
-	if outputQuality == "hd1080" {
-		fmt.Println("check ffmpeg is installed....")
-		ffmpegVersionCmd := exec.Command("ffmpeg", "-version")
-		if err := ffmpegVersionCmd.Run(); err != nil {
-			return fmt.Errorf("please check ffmpeg is installed correctly, err: %w", err)
+	if format.AudioChannels == 0 {
```

Comment:
or we check quality prefixed with `hd`?

Reply:
I prefer to check the prefix. I've added another commit to reflect this change.